linter/lints/cpcps/lint_crl_has_idp_test.go

```diff
@@ -5,11 +5,66 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/zmap/zlint/v3/lint"
+	"github.com/letsencrypt/boulder/linter/lints"
 
-	"github.com/letsencrypt/boulder/linter/lints/test"
+	lintTest "github.com/letsencrypt/boulder/linter/lints/test"
```

Comment:
this is a weird little nit, but I think we usually avoid capital letters / camelCase in our import aliases. I think this is because the go package names are always all-lowercase, so having an imported package with a `camelCase` name makes it look like not-a-package.